docs/RC/GetCapabilities/assets/GetCapabilities.txt

```diff
@@ -1,10 +1,10 @@
 participant App
 participant SDL
 participant HMI
-
-HMI->SDL:BC.OnReady
+App->SDL:RegisterAppInterface
```

Comment:
RC information is not included in the RAI response, it can only be retrieved by the app via `GetSystemCapability(REMOTE_CONTROL)`

Reply:
@jacobkeeler please see updates in ba730b1